integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java

```diff
@@ -464,6 +621,143 @@ public void testTreeNodeRefresh() throws InterruptedException {
 
   @Test
   @Order(20)
+  public void testViewFilesetCatalog() throws InterruptedException {
```

Comment:
`testViewFilesetCatalog` and `testViewKafkaCatalog` are best placed in the test process above.
I think we should avoid adding other operations in a single function test to make the test process more integrated.

Reply:
done